projects/kit/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -156,6 +169,10 @@ export class TuiInputPhoneInternationalComponent
 
     @tuiPure
     private calculateMaskAfterCountryCode(mask: string, countryCode: string): string {
-        return mask.replace(countryCode, '');
+        return mask.replace(countryCode, '').trim();
+    }
+
+    private getMaxAllowedLength(isoCode: TuiCountryIsoCode): number {
+        return COUNTRIES_MASKS[isoCode].replace(/[()\- ]/g, '').length;
```

Comment:
can we use `TUI_NON_DIGITS_REGEXP` here (for better readability) ?

Reply:
`TUI_NON_DIGITS_REGEXP` is not quite right, we need to # and + chars to count all length. But I can add new regexp variable, do u have any idea about naming?)

Reply:
it's difficult to find a common name for this group of characters.

It seems that you can leave it as it was =)